src/test/java/com/artipie/asto/key/KeyExcludeLastTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/asto/LICENSE.txt
+ */
+package com.artipie.asto.key;
+
+import com.artipie.asto.Key;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.IsEqual;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Test case for {@link KeyExcludeLast}.
+ *
+ * @since 1.9.1
+ */
```

Comment:
I'd suggest to add test when key for exclusion is not the last part because now it does not work in a proper way for this case

Reply:
@genryxy You are right !